styles/gmail/catppuccin.user.css

```
@@ -454,6 +454,44 @@
         background: lighten(@accent-color, 10%) !important;
       }
     }
+
+    /* loading */
+    #explosion_clipper_div > .la-i > div {
+      background: @surface0 !important;
+    }
+
+    #nlpt {
+        background-color: @surface0 !important;
+        ::before {
```

Apologies, I made a typo earlier.
```suggestion
        &::before {
```